index.html

```diff
@@ -6,20 +6,21 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="description" content="Website for the University of Waterloo NanoRobotics Group">
         <meta name="author" content="UW_NRG Controls">
-
         <title>UW_NRG</title>
         <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css"/>
         <link rel="stylesheet" type="text/css" href="css/style.css" />
     </head>
     <body>
-        <!-- <section id="loading_screen">
-            <div id="loading_bar"></div>
-            <div id="loading_item"></div>
-        </section> -->
         <nav class="navbar navbar-default navbar-fixed-top">
             <div class="container">
                 <div class="navbar-header">
-                    <a class="navbar-brand" id="logo" href="index.html"></a>
+                    <button  id="navbar-button" type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar">
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
```

Comment:
Could you clean this up a bit? It looks very weird right now. https://puu.sh/vSv58/1f5909bea7.png